### ISSUE TITLE
Jura — Redesign Cyrillic

### ISSUE THREAD

**Issue by alexeiva** [FontsToRaisins/alexei-log#3]

**Comment by alexeiva** [comment]
¶ #2. Warm-up, week 1, 2016-28-06
Updated Encoding project: switched from uni to nice names.
Started working on improving [or cleaning-up] the Jura font project. Migrated sources from SFD to UFO.
Original source files for the 4 weights were done without interpolation. Starting from Light, 
Bolder weight were added by offsetting the Light, and so on... Source files are in TT format.
As a result, no two masters are compatible, and the proportions and widths of characters are
inconsistent across styles. The charset is different in the 4 styles too.

Instead of fixing all this, I decided to work from scratch, and put the original file in the background for reference. 
By the end of the day I completed a single master in basic Latin a-z, and used Glyphs App filters (offset, round corners)
to generate 8 new styles programmatically. This was done as an exploration to see 
how far a single hairline master can be expanded. The Bolder weights of course require manual reworking. 
In overall the new approach seams viable, and much faster than fixing the master compatibility for each glyph.
